### DISCUSSION TITLE
Combine Carousel with Fancybox (tsx)

### DISCUSSION THREAD

**Discussion by Pierre31415** [fancyapps/ui#545] [Q&A]
Hello,
sorry I'm a newbie, it is surely very simple, but how do you combine a carousel in Facnybox (like at the bottom of this page  : https://fancyapps.com/carousel/) ? I use Typescript.
Is there a demo?

Thanks for your answer

**Comment by fancyapps** [comment]
Hi,

Actually, there are no extra steps, just initialize both components separately, here are some examples:

- ES6 - https://stackblitz.com/edit/js-vk4rms?file=index.js
- Vue - https://stackblitz.com/edit/vue-cuj9zw?file=src%2FApp.vue

**Comment by djuuuuus** [comment]
@Pierre31415 TY for raising this question which also actual for me too.
@fancyapps TY for the answer, if it's not difficult, could you please add this example to your getting started guide?
